sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -2181,8 +2277,10 @@ async def test_async_queue_send_large_message_receive(self, uamqp_transport, *,
                 await sender.send_messages(ServiceBusMessage(payload))
 
         # ReceiveMessages
-        async with ServiceBusClient.from_connection_string(
-                servicebus_namespace_connection_string, logging_enable=False, uamqp_transport=uamqp_transport) as sb_client:
+        fully_qualified_namespace = f"{servicebus_namespace.name}{SERVICEBUS_ENDPOINT_SUFFIX}"
```

Comment:
Passing in from the preparer might be better for future purposes, since it'll be easier to make any changes to this if needed.

Reply:
I see in your other comment that this will go in a another pr